pkg/kubelet/status/status_manager.go

```diff
@@ -431,7 +431,7 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 	normalizeStatus(pod, &status)
 	// The intent here is to prevent concurrent updates to a pod's status from
 	// clobbering each other so the phase of a pod progresses monotonically.
-	if isCached && isPodStatusByKubeletEqual(&cachedStatus.status, &status) && !forceUpdate {
+	if isCached && isPodStatusByKubeletEqual(&pod.Status, &status) && isPodStatusByKubeletEqual(&cachedStatus.status, &status) && !forceUpdate {
```

Comment:
is it ok that `updateLastTransitionTime ` will not update the transition time in this case?